### PR TITLE
Master nb11 rest service

### PR DESCRIPTION
**Pull request by algol60**
### Description of the Change
New run_plugins service. Also updated ExtractWordsFromTextPlugin.

### Alternate Designs
N/A

### Why Should This Be In Core?
That's where it is.

### Benefits
Multiple plugins can be run at once.

### Possible Drawbacks
Check with that ExtractWordsFromTextPlugin still works as expected.

### Verification Process
The run_plugins service was tested using a Jupyter notebook and ExtractWordsFromTextPlugin.
ExtractWordsFromTextPlugin unit tests were run, and more were added.

### Applicable Issues
N/A

### CODE REVIEW

File: CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RunPlugins.java [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+/**
+ * Run some plugins, possibly in parallel, optionally specifying a graph.
+ *
+ * @author algol
+ */
+@ServiceProvider(service=RestService.class)
+public class RunPlugins extends RestService {
+    private static final String NAME = "run_plugins";
+    private static final String GRAPHID_PARAMETER_ID = "graph_id";
```
Comment (cygnus-x-1) [inline CoreWebServer/src/au/gov/asd/tac/constellation/webserver/services/RunPlugins.java]: It hurts me that you have `GRAPHID_PARAMETER_ID` with the value `graph_id`...

File: CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/importing/ExtractWordsFromTextPlugin.java [hunk 2]
```diff
@@ -84,11 +85,16 @@
     public static final String REMOVE_SPECIAL_CHARS_PARAMETER_ID = PluginParameter.buildId(ExtractWordsFromTextPlugin.class, "ignore_special_chars");
     public static final String USE_REGEX_PARAMETER_ID = PluginParameter.buildId(ExtractWordsFromTextPlugin.class, "use_regex");
     public static final String WHOLE_WORDS_ONLY_PARAMETER_ID = PluginParameter.buildId(ExtractWordsFromTextPlugin.class, "whole_words_only");
-    public static final String CASE_INSENSITIVE_PARAMETER_ID = PluginParameter.buildId(ExtractWordsFromTextPlugin.class, "case_insensitive");
+    public static final String LOWER_CASE_PARAMETER_ID = PluginParameter.buildId(ExtractWordsFromTextPlugin.class, "case_insensitive");
```
Comment (cygnus-x-1) [inline CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/importing/ExtractWordsFromTextPlugin.java]: Also, `LOWER_CASE_PARAMETER_ID` = `case_insensitive`...

**Comment by algol60** [comment]
This is ready to be merged.

**Comment by arcturus2** [comment]
@algol60 are you at a stage where you would like this merged? Please advice when you can thank you!

**Pull request merged**